pandas/core/internals/blocks.py

```diff
@@ -1397,6 +1397,23 @@ def quantile(
         result = ensure_block_shape(result, ndim=2)
         return new_block_2d(result, placement=self._mgr_locs)
 
+    def round(self, decimals, using_cow=False) -> Block:
+        """
+        Rounds the values.
+        If the block is not of an integer or float dtype, nothing happens
+
+        Parameters
+        ----------
+        decimals: int,
+            Number of decimal places to round to.
+            Caller is responsible for validating this
+        using_cow: bool,
+            Whether Copy on Write is enabled right now
+        """
+        if not self.is_numeric or self.is_bool:
+            return self.copy(deep=None if using_cow else True)
```

Comment:
possibly raise?

Reply:
I believe that DataFrame.round() never raises even when there are no numeric columns, but Series.round() does.

Do you want me to add a comment that this doesn't raise?

Reply:
if this is just a perf PR than this should maintain behavior, but might merit an issue or TODO comment.  looks like some of our EAs have round methods that are not documented as part of the interface.